Docs/Book/Cookbook.md

```diff
@@ -339,7 +346,7 @@ Examples of using it:
 
 This produces:
 
-    0.0645003532293 0.168030378905 0.067523011904
+    0.0691236990428 0.111475253992 0.0716255058804
```

Comment:
More of a memo to myself: I should be sure that the doctests here are actually running and that they pass with this number of significant figures provided.